README.md

```diff
@@ -19,3 +19,90 @@ steps:
 ## License
 
 The scripts and documentation in this project are released under the [MIT License](LICENSE).
+
+## Development
+
+Once the dev container is started do the following:
+
+Log into Azure
+
+```bash
+az login
+az account set --subscription SUBSCRIPTION_ID
+```
+
+Run the npm installation
+
+```bash
+npm install
+```
+
+After changing the `index.js` file run either
+
+```bash
+npm run prepare
+```
+
+or use 
+
+```bash
+npm run dev
+```
+
+which will watch the file for changes and automatically compile the file.
+
+## Testing
+
+### With node
+
+To test the setup action an `.env.setup` file in the root directory with the following content
+
+```
```

Comment:
```suggestion
```ini
```

This will give good highlighting:


```ini
# Input overrides
INPUT_CONNECTION-STRING-NAME=OracleConnectionString
INPUT_TAG=setup-oracle-action
INPUT_INIT-SCRIPT=.github/workflows/scripts/init.sql
# Runner overrides
# Use LINUX to run on linux
RUNNER_OS=WINDOWS
RESOURCE_GROUP_OVERRIDE=yourResourceGroup
REGION_OVERRIDE=West Europe
```